.env-dist

```diff
@@ -1,27 +1,39 @@
+# dev, heroku, stage, production
 NODE_ENV=dev
-DISABLE_DOCKERFLOW=
 SERVER_URL=http://localhost:6060
 PORT=6060
-
 COOKIE_SECRET=3895d33b5f9730f5eb2a2067fe0a690e298f55f5e382c032fd3656863412
 
+# 1: disables the dockerflow endpoints
+# see: https://github.com/mozilla-services/Dockerflow#containerized-app-requirements
+DISABLE_DOCKERFLOW=
+
+# Database server
 DATABASE_URL="postgres://postgres@localhost:5432/blurts"
 
+# Email server
 SMTP_URL=""
-
+# From: address used in emails
 EMAIL_FROM=""
+# https://docs.aws.amazon.com/ses/latest/DeveloperGuide/using-configuration-sets.html
 SES_CONFIG_SET=""
+# username and password to authenticate messages coming back from SES
 SES_USERNAME=""
 SES_PASSWORD=""
-SES_NOTIFICATION_LOG_ONLY=""
+# 1: only log messages coming back from SES
+SES_NOTIFICATION_LOG_ONLY=
 
+# Firefox Accounts OAuth
 OAUTH_CLIENT_ID=cb1bef9d06bb9bc9
 OAUTH_CLIENT_SECRET=f5fb99de6e0af18ab17e013ac1d439903179a97a1c510fc10bc3bd50bbce089b
 OAUTH_AUTHORIZATION_URI="https://oauth-stable.dev.lcip.org/v1/authorization"
 OAUTH_PROFILE_URI="https://stable.dev.lcip.org/profile/v1/profile"
 OAUTH_TOKEN_URI="https://oauth-stable.dev.lcip.org/v1/token"
 
+# HIBP API for breach data
 HIBP_API_ROOT="https://stage.haveibeenpwned.com/api/v2/"
 HIBP_API_TOKEN=""
+HIBP_RELOAD_BREACHES_TIMER=60000
```

Comment:
Did you want the 1 minute timer, or 10 minutes?

Reply:
Doh, yeah I meant 10m. I changed the code in `controllers/hibp` to multiply this value by `1000` so this env var is in seconds now. Updated to 600 seconds.